### PR TITLE
Add graphql queries

### PR DESCRIPTION
**Pull request by duanxianbo**
[(task:20204)](https://staging.teamyapp.com/teams/107/tasks/20204)

### CODE REVIEW

File: src/client/appVersion.client.ts [hunk 1]
```diff
@@ -1,8 +1,13 @@
+import { print } from 'graphql';
 import { GraphQLClient } from '@lib/network/GraphQL.client';
 
 import { RemoteAppVersion } from '@core/client/entity/remoteAppVersion';
 
 import { RemoteUpdateAppVersionInput } from './entity/input';
+import createAppVersion from "./graphql/mutation/createAppVersion.graphql"
+import updateAppVersion from "./graphql/mutation/updateAppVersion.graphql"
+import finishAppPackageUploadSession from "./graphql/mutation/finishAppPackageUploadSession.graphql"
+
```
Comment (naturetime10) [inline src/client/appVersion.client.ts:10]:
```suggestion
```

File: src/client/graphql/mutation/createApp.graphql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (naturetime10) [inline src/client/graphql/mutation/createApp.graphql]:
```suggestion
```

File: src/client/user.client.ts [hunk 3]
```diff
@@ -10,6 +10,50 @@ export class UserClient {
         this.graphQLClient = graphQLClient;
     }
 
+
```
Comment (naturetime10) [inline src/client/user.client.ts]:
```suggestion
```

File: src/client/user.client.ts [hunk 4]
```diff
@@ -10,6 +10,50 @@ export class UserClient {
...
+    public getUser(userId: string): Promise<RemoteUser> {
+        return this.graphQLClient
+            .query({
+                query: `
```
Comment (naturetime10) [inline src/client/user.client.ts]: Should we move this to `.graphql` file?

File: src/client/user.client.ts [hunk 5]
```diff
@@ -10,6 +10,50 @@ export class UserClient {
...
+                            }
+                        }
+                    }
+                `,
+                variables: {
+                    userId,
+                },
+            })
+            .then((payload: any) => payload?.user);
+
```
Comment (naturetime10) [inline src/client/user.client.ts]:
```suggestion
```

File: src/global.d.ts [hunk 6]
```diff
@@ -2,3 +2,10 @@ declare function importModule(moduleUrl: string): Promise<any>;
 
 declare const BUILD_INJECT_FILE_PATH;
 declare const BUILD_INJECT_LINE_NUMBER;
+
+declare module '*.graphql' {
+    import { DocumentNode } from 'graphql';
+    const Schema: DocumentNode;
+
```
Comment (naturetime10) [inline src/global.d.ts]:
```suggestion
```

File: src/storage/syncer/stateSyncer.ts [hunk 7]
```diff
@@ -257,6 +257,14 @@ export class StateSyncer {
         this.localStore.updateState(appState);
     }
 
+    public async pullTeam(teamId: number) {
+        const remoteTeam = await this.teamClient.getTeamWithSprintPreviews(`${teamId}`);
+        let appState = this.localStore.getState();
+        appState = mergeTeam(appState, remoteTeam);
+        this.localStore.updateState(appState);
+    }
+
```
Comment (naturetime10) [inline src/storage/syncer/stateSyncer.ts:266]:
```suggestion
```

File: src/storage/syncer/stateSyncer.ts [hunk 8]
```diff
@@ -257,6 +257,14 @@ export class StateSyncer {
         this.localStore.updateState(appState);
     }
 
+    public async pullTeam(teamId: number) {
+        const remoteTeam = await this.teamClient.getTeamWithSprintPreviews(`${teamId}`);
+        let appState = this.localStore.getState();
```
Comment (naturetime10) [inline src/storage/syncer/stateSyncer.ts:262]: if (remoteTeam) {

File: src/storage/syncer/stateSyncer.ts [hunk 9]
```diff
@@ -892,6 +909,7 @@ export class StateSyncer {
     ): Promise<void> {
         const remoteApp = await this.appClient.createApp(`${teamId}`, input);
         let appState = this.localStore.getState();
+        debugger;
```
Comment (naturetime10) [inline src/storage/syncer/stateSyncer.ts]:
```suggestion
```

File: src/storage/syncer/stateSyncer.ts [hunk 10]
```diff
@@ -1080,6 +1098,7 @@ export class StateSyncer {
         appId: number,
         input: CreateStaticGroupInput,
     ): Promise<void> {
+        debugger
```
Comment (naturetime10) [inline src/storage/syncer/stateSyncer.ts]:
```suggestion
```

**Review by naturetime10 — CHANGES_REQUESTED** [review]

**Comment by naturetime10** [comment]
Please run `yarn format`

File: src/client/user.client.ts (hunk 4 above)
Comment (duanxianbo) [reply src/client/user.client.ts]: yes

**Review by naturetime10 — APPROVED** [review]

**Pull request merged**